Smarty/templates/Components/Components.tpl

```diff
@@ -206,3 +206,65 @@
 	{/foreach}
 	</ul>
 {/function}
+{literal}
```

Comment:
please move this javascript code to include/js/general.js so it gets cached and we can debug it